common/app/common/controllers/SunriseController.java

```diff
@@ -74,6 +74,7 @@ protected final SunrisePageData pageData(final UserContext userContext, final Pa
         final PageHeader pageHeader = new PageHeader(content.getAdditionalTitle());
         pageHeader.setLocation(new LocationSelector(projectContext(), userContext));
         pageHeader.setNavMenu(new NavMenuData(categories(), userContext, reverseRouter(), saleCategoryExtId));
+        pageHeader.setCustomerServiceNumber(configuration().getString("checkout.customerServiceNumber"));
```

Comment:
The reason why I didn't want to use this solution is because that means that every time the phone changes they have to call the developers to change it. I was rather opting for CMS, because it has to be something non-technical friendly.